packages/babel-plugin-proposal-optional-chaining/src/index.js

```diff
@@ -130,12 +141,14 @@ export default declare((api, options) => {
             if (!loose || !isSimpleMemberExpression(object)) {
               // memoize the context object in non-loose mode
               // `(a?.b.c)()` to `(a == null ? undefined : (_a$b = a.b).c.bind(_a$b))()`
-              baseRef = scope.maybeGenerateMemoised(object);
+              baseRef = scope.maybeGenerateMemoised(
+                skipTransparentExprWrappers(object),
```

Comment:
`object` is an ASTNode but not NodePath.

Reply:
Oh wow, thanks for catching that. I wonder why it works. Is .get() also defined on AST nodes?

Will look at this tomorrow, I think I might need a new function in the helper package.

Reply:
Sorted :)